edx_sga/utils.py

```diff
@@ -9,10 +9,33 @@
 
 import pytz
 from django.conf import settings
-from django.core.files.storage import default_storage
+from django.core.files.storage import default_storage as django_default_storage, get_storage_class
 from edx_sga.constants import BLOCK_SIZE
 
 
+def get_default_storage():
```

Comment:
Can you write a simple test for this? That would go in [test_utils.py](https://github.com/mitodl/edx-sga/blob/master/edx_sga/tests/test_utils.py).

Reply:
Added